domains/ohxxm.json

```diff
@@ -0,0 +1,19 @@
+{
+    "owner": {
+        "username": "ohxxm",
+        "email": "ohmrc2545@gmail.com"
+    },
+    "record": {
+        "CNAME": "ohxxm.github.io"
+        "A": [
+            "185.199.108.153",
+            "185.199.109.153",
+            "185.199.110.153",
+            "185.199.111.153"
+        ],
```

Comment:
Cant use CNAME with any other record

Reply:
mb im sped when it comes to cname and records.